stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -441,6 +443,48 @@ extension ExecutorJob {
   }
 }
 
+// Helper to create a trampoline job to execute a job on a specified
+// executor.
+extension ExecutorJob {
+
+  /// Create a trampoline to enqueue the specified job on the specified
+  /// executor.
+  ///
+  /// This is useful in conjunction with the `Clock.run()` API, which
+  /// runs a job on an unspecified executor.
+  ///
+  /// Parameters:
```

Comment:
```suggestion
  /// - Parameters:
```

Needs the `-` I think?